tests/unit/js_utils/text_validation.spec.js

```diff
@@ -116,22 +116,23 @@ describe("TextValidation.validate_plate_barcode", () => {
     ["MA20044", "7"],
     ["MA20**#*", 8],
   ])(
-    "When an improper platecode text  %s with the [input.length = %] and special characters as its not matching criteria, Then validation fails and feedback text is <space>",
+    "When an improper platecode text  %s with the [input.length = %s] and special characters as its not matching criteria, Then validation fails and feedback text is <space>",
     (plate_bar_code, error) => {
       const text = plate_bar_code;
       const TestPlateBarCode = TextValidation_PlateBarcode;
       expect(TestPlateBarCode.validate(text)).toStrictEqual(" ");
     }
   );
   test.each([
-    ["MA20044001", "10"],
-    ["M120044099", "11"],
+    ["MA20044001", "10", ""],
+    ["M120044099", "11", " "], // M1 is invalid so returns space
```

Comment:
As noted in earlier comment, include a description of what is right or wrong about the barcode in the array of test parameters

Reply:
Included the information of `MA is still valid`, `M1 disallowed` and `MD is new valid value`.